cmd/zoekt-indexserver/config.go

```diff
@@ -222,68 +207,75 @@ func periodicMirrorFile(ctx context.Context, repoDir string, opts *Options, pend
 }
 
 func createGithubArgsMirrorAndFetchArgs(ctx context.Context, c ConfigEntry) []string {
-	// work begins
-	ctx, span := tracer.Start(ctx, "createGithubArgsMirrorAndFetchArgs")
-
-	// end span once done with func
-	defer span.End()
+	trace, ctx := internalTrace.New(ctx, "zoekt-indexserver.createGithubArgsMirrorAndFetchArgs", "")
+	defer trace.Finish()
 
 	args := make([]string, 0)
 	if c.GitHubURL != "" {
 		args = append(args, "-url", c.GitHubURL)
-		span.SetAttributes(attribute.Key("url").String(c.GitHubURL))
+		trace.LogFields(
+			openTracingLog.String("url", c.GitHubURL),
+		)
```

Comment:
None of these should be here in `createGithubArgsMirrorAndFetchArgs` - remember `loggedRun` prints these args out?